specs/enable-selinux.adoc

```diff
@@ -0,0 +1,197 @@
+= Enable SELinux for tendrl
+
+SELinux should be enabled in tendrl for the system which controled by selinux.
```

Comment:
Sure, we suggest selinux enforcing for gluster nodes also.
I hope gluster have its own selinux policy.